apps/prairielearn/elements/pl-dropdown/pl-dropdown.py

```diff
@@ -56,6 +56,12 @@ def prepare(element_html, data):
         required_attribs=["answers-name"],
         optional_attribs=["blank", "allow-blank", "weight", "sort"],
     )
+
+    if pl.has_attrib(element, "allow-blank") and not pl.has_attrib(element, "blank"):
+        raise ValueError(
+            'The attribute "allow-blank" cannot be enabled when blank dropdown entries are disabled by the "blank" attribute.'
+        )
```

Comment:
You're checking if the attribute was set, not the value it was set to. `allow-blank="false"` should not cause this error. You probably want `pl.get_boolean_attrib`, not `pl.has_attrib`.

Reply:
Right. *Now* it should work as intended (this time I've also tested that the error gets triggered as expected)